python/beeai_framework/backend/chat.py

```diff
@@ -39,89 +46,6 @@
 logger = Logger(__name__)
 
 
-class ChatModelParameters(BaseModel):
```

Comment:
this is also breaking change..

Reply:
technically not because they are still being exported the same, though that may change when we deal with public/private cleanup